docs/blog/oras-0.15-a-fully-functional-registry-client.md

```diff
@@ -0,0 +1,189 @@
+# ORAS 0.15: A fully Functional OCI Registry Client
+
+_Author: [Feynman Zhou](https://twitter.com/FeynmanZhou), [Yi Zha](https://github.com/yizha1)_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced two releases of v0.15 for the ORAS CLI recently. ORAS [v0.15.0](https://github.com/oras-project/oras/releases/tag/v0.15.0) introduces four new top-level commands and new options to manage tags and repos in registries and provides capabilities to alter the OCI content for advanced use cases. Three weeks later, [ORAS 0.15.1](https://github.com/oras-project/oras/releases/tag/v0.15.1) also released with a few known bug fixes. Since the release of v0.15, ORAS CLI has evolved into a fully functional OCI registry client.
+
+## What's new in ORAS 0.15
+
+![ What's new in ORAS 0.15](oras-0.15/what's-new-in-oras-0.15.png)
+
+As ORAS has been adopted by more and more OCI implementors and registry vendors, we have seen increased community requirements in providing fine-grained capabilities to alter the content of OCI supply chain artifacts. ORAS 0.15 now supports operate the blob and manifest of an artifact in the registry. Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.15.0) for details.
+
+This blog post will demonstrate how to use ORAS CLI v0.15 to convert a Docker image stored in Docker Hub into an OCI image， then push it to the Distribution registry.
```

Comment:
I understand the point of demonstrating a scenario by using new ORAS commands, but there are some commands not covered, for example `manifest delete`, `blob delete`, `manifest fetch-config`, and `repo list`.

Reply:
Good question. The purpose is to solve an issue using most of the new commands in ORAS 0.15. It's hard to cover all commands in a single demo. I will state this demo just leveraged most of the new commands in this blog.

Reply:
> Recommend using the alias everywhere since we want the blog post to showcase the ease of use. The long commands are present in the official documentation and I hopefully we will be moving to the short commands as the actual command in 0.16 anyway.

Agree. We will use short commands in the command line by default.